app/views/trainees/start_dates/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render PageTitle::View.new(title: "trainees.trainee_ids.edit") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render DynamicBackLink::View.new(@trainee, text: t(:back_to_record)) %>
+<% end %>
+
+<%= form_with model: @training_details, url: trainee_start_date_path, local: true do |f| %>
+  <%= f.govuk_date_field :commencement_date, legend: {
+    text: t("views.forms.training_details.commencement_date.label"), size: 's'
+  }, hint: { text: t("views.forms.training_details.commencement_date.hint") + " 9 2 2021" } %>
```

Comment:
```suggestion
  }, hint: { text: t("views.forms.training_details.commencement_date.hint") + " 15 2 2021" } %>
```

Reply:
What's the benefit of having the hint stored in two places?

Reply:
It's only store in one place but used in 2 places, namely `app/views/trainees/start_dates/edit.html.erb` and `app/views/trainees/training_details/edit.html.erb`

Reply:
@jhenderson no I meant that the hint is compiled from two sources - the date gets appended on.